sepolicy/untrusted_app.te

```diff
@@ -1 +1,2 @@
 allow untrusted_app storage_stub_file:dir getattr;
+allow untrusted_app baseband_prop:file r_file_perms;
```

Comment:
Do we need to give untrusted apps access too? Others seem fine, but I don't think this one is needd. Correct me if I'm wrong.

Reply:
As i understand, a typical app is considered an unstrusted app.
Some apps (like the ones who show hardware/software info) read the baseband version.

Reply:
Aren't those supposed to access it using system APIs?

Reply:
Fixed on last commit. Don't worry about having two commits, as you can squash it.

Reply:
This one actually should be fine as 2 commits, so we see it was intentionally disabled